src/interactive-window/interactiveWindow.ts

```diff
@@ -177,7 +181,6 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
         return editor;
     }
 
-    @chainable()
```

Comment:
Fix from https://github.com/microsoft/vscode-jupyter/pull/9458

Brought it here to have the tests pass properly and so that I can write more tests in this PR